e2e/playwright/text-to-cad-tests.spec.ts

```diff
@@ -689,3 +691,749 @@ test(
+
+      // Make sure we went back home
+      await expect(
+        page.getByText('No Projects found, ready to make your first one?')
+      ).toBeVisible()
+    }
+  )
+
+  test(
+    'Home Page -> Text To CAD -> New Project -> Navigate to the project -> Accept -> should stay in same file',
+    { tag: '@electron' },
+    async ({ context, page }, testInfo) => {
+      const projectName = 'my-project-name'
+      const prompt = '2x2x2 cube'
+      await mockPageTextToCAD(page)
+
+      // open commands
+      await page.getByTestId('command-bar-open-button').click()
+
+      // search Text To CAD
+      await page.keyboard.type('Text To CAD')
+      await page.keyboard.press('Enter')
+
+      // new project
+      await page.keyboard.type('New project')
+      await page.keyboard.press('Enter')
+
+      // write name
+      await page.keyboard.type(projectName)
+      await page.keyboard.press('Enter')
+
+      // prompt
+      await page.keyboard.type(prompt)
+      await page.keyboard.press('Enter')
+
+      // Go into the project that was created from Text to CAD
+      await page.getByText(projectName).click()
+
+      await page.getByRole('button', { name: 'Accept' }).click()
+
+      await expect(page.getByTestId('app-header-project-name')).toBeVisible()
+      await expect(page.getByTestId('app-header-project-name')).toContainText(
+        projectName
+      )
+      await expect(page.getByTestId('app-header-file-name')).toBeVisible()
+      await expect(page.getByTestId('app-header-file-name')).toContainText(
+        '2x2x2-cube.kcl'
+      )
+    }
+  )
+
+  test(
+    'Home Page -> Text To CAD -> Existing Project -> Navigate to the project -> Reject -> should load main.kcl',
+    { tag: '@electron' },
+    async ({ context, page }, testInfo) => {
+      const u = await getUtils(page)
+      const projectName = 'my-project-name'
+      const prompt = '2x2x2 cube'
+      await mockPageTextToCAD(page)
+
+      // Create and navigate to the project then come home
+      await createProject({ name: projectName, page, returnHome: true })
+
+      await expect(page.getByText('Your Projects')).toBeVisible()
+
+      // open commands
+      await page.getByTestId('command-bar-open-button').click()
+
+      // search Text To CAD
+      await page.keyboard.type('Text To CAD')
+      await page.keyboard.press('Enter')
+
+      // new project
+      await page.keyboard.type('Existing project')
+      await page.keyboard.press('Enter')
+
+      // write name
+      await page.keyboard.type(projectName)
+      await page.keyboard.press('Enter')
+
+      // prompt
+      await page.keyboard.type(prompt)
+      await page.keyboard.press('Enter')
+
+      // Go into the project that was created from Text to CAD
+      // This only works because there is only 1 project. Each project has the same value of `data-test-id='project-title'`
+      await page.getByTestId('project-title').click()
```

Comment:
In future, this should make use of the `homePage.openProject` fixture method like this example here https://github.com/KittyCAD/modeling-app/blob/main/e2e/playwright/feature-tree-pane.spec.ts#L82-L82